packages/ws-worker/src/server.ts

```diff
@@ -135,6 +135,12 @@ function connect(app: ServerApp, logger: Logger, options: ServerOptions = {}) {
     logger.debug(e);
   };
 
+  // handles messages for the worker:queue
+  const onMessage = (event: string) => {
+    if (event === WORK_AVAILABLE)
+      claim(app, logger, { maxWorkers: options.maxWorkflows });
```

Comment:
Oh this is the error thing right?

So a couple of things here:
1) `claim` throws a lot. That's fine. It's designed to throw if it didn't get any work back, and this response informs the backoff.
2) But if `claim` throws here, we should catch and ignore it and do nothing.
3) Also note that the claim call here does not affect the backoff loop in anyway. It'll claim if the loop is closed. It won't reset or increase the backoff. 
4) When the claimed run is completed, the only impact on the backoff loop is: if the loop has stopped (presumably because we're at capacity), then restart it and make a claim at `minBackoff`. If we have capacity, and the loop is alive, it'll be unaffected by the completion of the run.

I think this is all perfectly okay.

Reply:
error?